server/config/base.js

```diff
@@ -138,7 +138,18 @@ module.exports = {
             propertyToMatch: 'url2',
             fieldDescription: 'test trait'
         }
-    ]
+    ],
+    
+    tagValuesTransformMap: {
+        success: {
+            type: 'boolean'
```

Comment:
Here, all tags with `key` `success` will be transformed according to the configuration in the `success` object. The `type` property of `success` refers to which transform function will be used (in this case `booleanTransform`). There are currently two types, `boolean` or `link`. Otherwise, there is no need to configure the tag as default transform will be used.